### DISCUSSION TITLE
Unable to verify Github pages, custom domain

### DISCUSSION THREAD

**Discussion by LawanIdriss** [community/community#30192] [Other Feature Feedback, Questions, & Ideas]
Hi

 everyone!, I'm looking for help adding custom domain to my GitHub pages, the domain is verified on account level, but failed in repository level. I'm using GitHub Pro and has another domain that is working fine but this one is not working.

I will be glad if you will me help fix the issues.

Screenshots attached.
![Screen1](https://user-images.githubusercontent.com/37778249/185742616-a7ab91e8-5fcc-4370-92ca-57eabe866d86.png)
![Screen2](https://user-images.githubusercontent.com/37778249/185742623-d56c0f5a-ef2a-449f-9e03-c8693cac4d69.png)

**Comment by airtower-luna** [comment]
Verification uses a special TXT record to prove that you control the domain, by showing you can set the TXT record.

To actually *use* the domain you still need to add A records to point at certain GitHub servers, as described here: https://docs.github.com/en/pages/configuring-a-custom-domain-for-your-github-pages-site/managing-a-custom-domain-for-your-github-pages-site#configuring-an-apex-domain

Right now there doesn't seem to be *any* A record for your domain.
